yarn.lock

```diff
@@ -318,6 +318,38 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@automattic/calypso-stats@workspace:apps/stats":
+  version: 0.0.0-use.local
+  resolution: "@automattic/calypso-stats@workspace:apps/stats"
+  dependencies:
+    "@automattic/calypso-apps-builder": "workspace:^"
+    "@automattic/calypso-babel-config": "workspace:^"
+    "@automattic/calypso-build": "workspace:^"
+    "@automattic/calypso-polyfills": "workspace:^"
+    "@automattic/webpack-extensive-lodash-replacement-plugin": "workspace:^"
+    "@automattic/webpack-inline-constant-exports-plugin": "workspace:^"
+    autoprefixer: ^10.2.5
+    calypso: "workspace:^"
+    enzyme: ^3.11.0
+    html-webpack-plugin: ^5.0.0-beta.4
+    jest: ^27.2.4
+    page: ^1.11.6
```

Comment:
Main Calypso is still on `page@1.11.5`, let's be consistent with that, or upgrade `page` for the entire project. Then we won't get a duplicate `page@npm:^1.11.6` record in `yarn.lock`.

Reply:
Resolved in [f1e241a](https://github.com/Automattic/wp-calypso/pull/69231/commits/f1e241aaed02f7c110cc5496b369ed2e06657197)